### PR TITLE
Test "enableIdn" option with no intl extension available (Email, Url rules)

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #202

### CODE REVIEW

File: src/Rule/Email.php [hunk 1]
```diff
@@ -5,13 +5,11 @@
 namespace Yiisoft\Validator\Rule;
```
Comment (arogachev) [inline src/Rule/Email.php:5]: It was decided to use CI instead.

**Comment by xepozz** [comment]
Could it be done with https://github.com/yiisoft/validator/blob/ea9d5279bed2f0213efed8df692ba241df23b0b3/tests/bootstrap.php#L8?

**Comment by arogachev** [comment]
@xepozz Nope, it couldn't. Checked it again. This method only works once for Url rule. Already faced this problem.

**Comment by arogachev** [comment]
Closing in favor of #277.

**Pull request closed without merging**